public/app/common/components/controls/button/button.jsx

```diff
@@ -16,8 +16,13 @@ module.exports = React.createClass({
     });
 
     return (
-      <a className={ className } href='#' onClick={this.props.onClick}> { this.props.children }</a>
```

Comment:
It might be to default event handling prevention in the button component so that higher order consuming components don't have to worry about that